nms/server/alerts/sync.ts

```diff
@@ -11,22 +11,22 @@
  * limitations under the License.
  *
  */
-import MagmaAPI from '../../api/MagmaAPI';
+import OrchestratorAPI from '../../api/MagmaAPI';
 import getCwfAlerts from './cwfAlerts';
 import getFegAlerts from './fegAlerts';
 import getLteAlerts from './lteAlerts';
 
```

Comment:
Did we discuss, if we only wanted to have one combined import block?

Reply:
Cleaning up the imports is a separate thing imho.